rubyforgood/human-essentials
# 3732 - Extract Donations (All Sources) from Dashboard

Resolves #3732

### Description
We want to extract the "Donations (All Sources)" card from the dashboard and move it to a Donations Summary page.

### Type of change



* Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?

- Go to `/reports/donations_summary`
- See all info that was previously available in the dashboard

Tested manually, and tested auth using request specs, and tested using system specs.

### Screenshots
![image](https://github.com/rubyforgood/human-essentials/assets/8124558/325c0d17-e54c-49cf-8b8f-8be220e0bca4)
